src/services

```diff
@@ -712,3 +712,27 @@ p({
   "serve": true,
   "serviceClass": "foam.nanos.session.SimpleSessionService"
 })
+p({
+  "class":"foam.nanos.boot.NSpec",
+  "name":"currencyDAO",
+  "serve":true,
+  "serviceScript":
+  """
+  return new foam.dao.EasyDAO.Builder(x)
```

Comment:
This block from lines 721-729 should be indented two more spaces. Same goes for lines 733-736 below.

Reply:
addressed.